_includes/homepage/hero.html

```diff
@@ -1,105 +1,121 @@
-                            </a>
-                        {%- endif -%}
-                    {%- endif -%}
-                </div>
+  <div class="bp-hero-body">
+    <div class="bp-container margin--top--lg">
+      <div class="row is-vcentered is-centered ma">
+        {%- if section.hero.variant != 'image' -%}
+        <div class="col is-9 has-text-centered has-text-white">
+          <h1 class="display padding--bottom--lg margin--none">
+            <b class="is-hidden-touch">{{- section.hero.title -}}</b>
+            <b class="is-hidden-desktop">{{- section.hero.title -}}</b>
+          </h1>
+          {%- if section.hero.subtitle -%}
+          <p class="is-hidden-mobile padding--bottom--lg">
+            {{- section.hero.subtitle -}}
+          </p>
+          {%- endif -%} {%- if section.hero.dropdown -%}
+          <div class="bp-dropdown margin--top--sm">
+            <div class="bp-dropdown-trigger">
+              <a
+                class="bp-button bp-dropdown-button hero-dropdown is-centered"
+                aria-haspopup="true"
+                aria-controls="hero-dropdown-menu"
+              >
+                {%- if section.hero.dropdown.title -%}
+                <span>
+                  <b>
+                    <p>{{- section.hero.dropdown.title -}}</p>
+                  </b>
+                </span>
+                {%- else -%}
+                <span>
+                  <b>
+                    <p>I want to...</p>
+                  </b>
+                </span>
+                {%- endif -%}
+                <span class="icon is-small">
+                  <i
+                    class="sgds-icon sgds-icon-chevron-down is-size-4"
+                    aria-hidden="true"
+                  ></i>
+                </span>
+              </a>
             </div>
+            <div
+              class="bp-dropdown-menu has-text-left"
+              id="hero-dropdown-menu"
+              role="menu"
+            >
+              <div class="bp-dropdown-content is-centered">
+                {%- for option in section.hero.dropdown.options -%} {%- if
+                option.url and option.title -%}
+                <a
+                  class="bp-dropdown-item"
+                  href="{{- site.baseurl -}}{{ option.url }}"
+                >
+                  <h5>{{ option.title }}</h5>
+                </a>
+                {%- endif -%} {%- endfor -%}
+              </div>
+            </div>
+          </div>
+          {%- else -%} {%- if section.hero.url and section.hero.button -%}
+          <a
+            href="{{- site.baseurl -}}{{- section.hero.url -}}"
+            class="bp-button is-secondary is-uppercase search-button"
+          >
+            {{- section.hero.button -}}
+          </a>
+          {%- endif -%} {%- endif -%}
         </div>
+        {%- endif -%} {%- if section.hero.variant == 'image' -%}
+        <div class="min-height-mobile" />
+        {%- endif -%}
```

Comment:
only new thing added here - rest are formatting

Reply:
Nit checking: Any reason you separated the if-clause? I was thinking it might be easier to follow the conditionals if we had something like this:

```
{%- if section.hero.variant == 'image' -%}
...min-height...
{%- else -%}
...existing code...
{%- endif -%}
```

We can also leave it open to future `elsif` when we get more variants.

Reply:
cos mutually exclusive - for now, at least. next time (next week) will have more so maybe then?